randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -117,21 +124,41 @@ Extra - asset_id: 3
   * Layers: default
   * Normal Door to Crateria Kihunter Room/Door to Moat
   > Door to West Ocean
-      Grapple Beam or Space Jump
+      Any of the following:
+          Grapple Beam or Space Jump
+          Speed Booster and Short Charge (Intermediate) and Disabled Entrance Randomizer
+          Gravity Suit and Gravity Jump (Intermediate) and Knowledge (Intermediate)
+          All of the following:
+              # Using a CWJ to get across
+              Knowledge (Intermediate) and Movement (Advanced) and Wall Jumps (Expert)
+          All of the following:
+              # HBJ across
+              Horizontal Infinite Bomb Jump (Advanced) and Can Lay Bombs
   > Pickup (Missile Expansion)
-      Grapple Beam or Space Jump
+      Any of the following:
+          Grapple Beam or Space Jump or Movement (Beginner)
+          Gravity Suit and Wall Jumps (Beginner)
+          Speed Booster and Short Charge (Intermediate) and Disabled Entrance Randomizer
 
 > Door to West Ocean; Heals? False
   * Layers: default
   * Normal Door to West Ocean/Door to Moat
   > Door to Crateria Kihunter Room
-      Grapple Beam or Space Jump
+      Any of the following:
+          Any of the following:
+              # Via top path - TODO: check for shinespark without entrance rando?
+              Grapple Beam or Space Jump
+          Any of the following:
+              # Via bottom path
+              Screw Attack or Can Lay Bombs
```

Comment:
Do you need gravity suit to screw attack underwater? That might be a Fusion thing.

Reply:
Yeah I think you have to be out of water first. Also lol Fandom being way out of date since it doesn't include msr or dread

Reply:
yeah metconst confirmed